docs/reference/data-streams.asciidoc

```diff
@@ -0,0 +1,115 @@
+[chapter]
+[[data-streams]]
+= Data Streams
+
+You can use data streams to index time-based data that's continuously generated.
+A data stream groups indices from the same time-based data source.
+A data stream tracks its indices, known as _backing indices_, using an ordered
+list.
+
+A data stream's backing indices are <<index-hidden,hidden>>.
+The most recently created backing index is the
+write index. The other backing indices handle read requests. A data stream only
```

Comment:
Minor clarification: Read requests are handled by all backing indices, not just the ones that are not the write index.